app/components/post_list/post/index.ts

```diff
@@ -96,7 +96,7 @@ const withSystem = withObservables([], ({database}: WithDatabaseArgs) => ({
 
 const withPost = withObservables(
     ['currentUser', 'isCRTEnabled', 'post', 'previousPost', 'nextPost'],
-    ({appsEnabled, currentUser, database, isCRTEnabled, post, previousPost, nextPost}: PropsInput) => {
```

Comment:
`appsEnabled` is not used in this function. It should properly fall through if not looked at here, so I directly removed it.